src/mmw/js/src/modeling/tests.js

```diff
@@ -693,14 +693,17 @@ describe('Modeling', function() {
                     });
 
                     model.updateModificationHash();
-                    assert.equal(model.get('modification_hash'), '582b6178186440159bd4ea25f0260892');
+                    assert.equal(model.get('modification_hash'),
+                                 'ca1872c7647c7cb9a7f0a00b683407df' );
```

Comment:
I'm having to revert these three changes to get the tests to pass

Reply:
I had to adjust these to get it to pass on my local, but Jenkins wanted them changed. I've pushed a fixup to revert.